charts/move2kube/README.md

```diff
@@ -0,0 +1,21 @@
+Move2kube
+===========
+
+# Installation
+From `charts` folder run 
+```console
+helm install move2kube move2kube
+```
+
+Then update `move2kube_url` in [values file](values.yaml) with the route of the move2kube instance:
```

Comment:
instead, you can update the helm command to provide the value with adding the `--set=...` instead of editing the values.yaml file.

Reply:
you mean run directly `helm upgrade move2kube move2kube --set=move2kube_url:<url>` ?